packages/web/components/MegaMenu/DesktopPlayerStats.tsx

```diff
@@ -41,37 +43,94 @@ export const DesktopPlayerStats: React.FC<PlayerStatsProps> = ({ player }) => {
             _hover={{ transform: 'scale(0.9)' }}
           />
         </MenuButton>
-        <MenuList mt="8px" color="black">
+        <MenuList
+          mt="8px"
+          color="white"
+          boxShadow="2xl"
+          bg="linear-gradient(180deg, rgba(42, 31, 71, 0.9) 6.18%, rgba(17, 3, 32, 0.86) 140%)"
+          borderRadius="md"
+          border={0}
+          px={2}
+          position="relative"
+        >
+          <Icon
+            position="absolute"
+            left="calc(92% - 1.25rem)"
+            top={-3}
+            w={6}
+            h={'auto'}
+            opacity={1}
+            transition="opacity 0.2s"
+            zIndex={2}
+          >
+            <path
```

Comment:
Were I doing this, I'd put this icon in a file under `design-system/src/icons` and give it a name since there's no way to tell what this icon is for from the `path`.

Reply:
Addressed in my [latest commit](https://github.com/MetaFam/TheGame/pull/1370/commits/a7cef79ed03fbb8ce82790e5d7294ed8f6ccd310)